src/assets/styles/smp.scss

```diff
@@ -22,6 +23,7 @@
   }
 }
 
+// stylelint-disable-next-line selector-class-pattern
```

Comment:
It would be best if we don't put lots of lint-error-suppressors in the code and instead address the cause. It looks to me that turning this into a kebab-case classname would resolve this issue (and tbh would bring it more in line with css class names elsewhere in the code).

Reply:
This class is set by SMP - we have no control over it